caffe2/core/hip/context_hip.h

```diff
@@ -51,7 +52,7 @@ class ThreadLocalHIPObjects {
 
   hipStream_t GetStream(int gpu, int stream_id) {
     vector<hipStream_t>& gpu_streams = hip_streams_[gpu];
-    if (gpu_streams.size() <= stream_id) {
+    if (gpu_streams.size() <= (unsigned)stream_id) {
```

Comment:
Can we use static_cast<> here?

Reply:
We can, I was just imitating the cuda file.